test/scripts/kind_dual_conversion.sh

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/env bash
+set -x
+
+convert_kubelet() {
+ echo "Converting kubelet on nodes $NODES"
+ for n in $NODES; do
+  # /var/lib/kubelet/config.yaml
+  # cat <<EOF >> "/var/lib/kubelet/config.yaml"
+# featureGates:
+#   IPv6DualStack: true
+# EOF
+   docker exec $n cp /var/lib/kubelet/config.yaml /var/lib/kubelet/config.yaml.bak
+   docker exec $n sed -i '/featureGates/a\  IPv6DualStack: true' /var/lib/kubelet/config.yaml
+   # Restart kubelet
+   docker exec $n systemctl restart kubelet
+  echo "Converted kubelet on node $n"
+ done
+}
+
+convert_cni() {
+  # from kind hack/ci/e2e-k8s.sh
+  # Modify current OVN config
+  original_ovnconfig=$(kubectl get -oyaml -n=ovn-kubernetes configmap/ovn-config)
+  echo "Original OVN config:"
+  echo "${original_ovnconfig}"
+  # Patch it
+  fixed_ovnconfig=$(
+    printf '%s' "${original_ovnconfig}" | sed \
+      -e "s#^  net_cidr.*#&\,${SECONDARY_CLUSTER_SUBNET}#" \
+      -e "s#^  svc_cidr:.*#&\,${SECONDARY_SERVICE_SUBNET}#" \
+  )
+  echo "Patched OVN config:"
+  echo "${fixed_ovnconfig}"
```

Comment:
you wanna cat the file here for easier debugging? I guess, same comment for the kubelet config above too.

Reply:
this already prints the modified config, no need to cat :)

Reply:
ummm... yeah it does. ha. not sure how I missed that 😃